src/components/InputCurrency/index.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { TextField, InputAdornment } from '@material-ui/core';
+
+export const parseEventValueToNumberIfValueExist = event => {
+  let result = '';
+
+  if (event && event.target && event.target.value) {
+    result = parseFloat(event.target.value);
+  }
+
+  return result;
+}
+
+const InputCurrency = ({ value, label, minimum, errors, onChange, currencySymbol, focused, ...props }) => {
+  if ((value !== undefined && isNaN(value)) || (minimum !== undefined && value < minimum)) {
+    return null
+  }
+
+  const handleChange = event => {
+    const { target: { value: eventValue } } = event;
+
+    if (onChange) {
+      if (
+        (minimum === undefined || value === undefined) ||
```

Comment:
- minimum === undefined pode ser !minimum

Reply:
Quando for 0 ele passa nessa condição, deixei explícito pra evitar que o 0 passasse